lib/LWP/UserAgent.pm

```diff
@@ -1133,22 +1133,48 @@ LWP::UserAgent - Web user agent class
 
 =head1 SYNOPSIS
 
- use strict;
- use warnings;
- use LWP::UserAgent ();
+    use strict;
+    use warnings;
 
- my $ua = LWP::UserAgent->new;
- $ua->timeout(10);
- $ua->env_proxy;
+    use LWP::UserAgent ();
 
- my $response = $ua->get('http://example.com');
+    my $ua = LWP::UserAgent->new(timeout => 10);
+    $ua->env_proxy;
+
+    my $response = $ua->get('http://example.com');
+
+    if ($response->is_success) {
+        print $response->decoded_content;    # or whatever
+    }
+    else {
+        die $response->status_line;
+    }
+
+Extra layers of security (note the C<cookie_jar> and C<protocols_allowed>):
+
+    use strict;
+    use warnings;
+
+    use HTTP::CookieJar::LWP ();
+    use LWP::UserAgent       ();
 
- if ($response->is_success) {
-     print $response->decoded_content;  # or whatever
- }
- else {
-     die $response->status_line;
- }
+    my $jar   = HTTP::CookieJar::LWP->new;
+    my $agent = LWP::UserAgent->new(
```

Comment:
s/$agent/$ua/

Also, I think we should use the same variable name for LWP::UserAgent objects in this document.
So why don't you use `$ua` in "BEST PRACTICES" section too?

Reply:
Thanks, I've fixed all of those cases now.  This was partly a copy/paste from `WWW::Mechanize`, which is where `$agent` came from.  Although, looking more closely at that, it should probably have been `$mech` in the `WWW::Mechanize` docs, since that's what everything else there uses.